storage/tables/index.md

```diff
@@ -255,15 +255,13 @@ imported row was completely identical to the existing one). Therefore,
 when using incremental processing set to `1 day`, you will not receive that row in input mapping.
 
 
-#### Adaptive Incremental Processing
+#### Automatic Incremental Load

```

Comment:
to je blby, Incremental Load je dovnitr KBC, az to bude jednou v transformacich, tak to nebude davat smysl

Reply:
A jak bys to pojmenoval? I vzhledem k UI...

Reply:
a je neco spatnyho na incremental processing?

Reply:
vzhledem k UI by se ten predchozi odstavec mel jmenovat Manual incremental processing a tento Automatic incremental processing a mely by byt prehozeny

Reply:
hmhm, ok!

Reply:
to tak nejde

Reply:
?

Reply:
predelal jsem to, mrkni na to

Reply:
jo, tak to je super!